dll/win32/shell32/shlexec.cpp

```diff
@@ -1847,6 +1900,12 @@ static WCHAR *expand_environment( const WCHAR *str )
     return buf.Detach();
 }
 
+static inline BOOL
+PathIsShortcutFile(LPCWSTR pszFileName)
+{
+    return lstrcmpiW(PathFindExtensionW(pszFileName), L".LNK") == 0;
```

Comment:
```suggestion
    return lstrcmpiW(PathFindExtensionW(pszFileName), L".lnk") == 0;
```
(rest of our source code seems to prefer using lowercase extensions: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=%5C%22%5C.lnk%5C%22&sr=1 )